lerna.json

```diff
@@ -7,9 +7,6 @@
     }
   },
   "npmClient": "yarn",
-  "packages": [
-    "packages/*"
```

Comment:
is it ok to remove les package path for lerna ?

Reply:
lerna is directly using the workspace described in the `package.json`
This change could be a separate pr